onboarding-scripts/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=onboarding-scripts
+#PKG_VERSION:=dev
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
PKG_BUILD_DIR is defined by default as: $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION).